test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -801,67 +801,135 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
 
     #endregion Redirect tests
 
-    #region SkipHeaderVerification Tests
+    Context "Invoke-WebRequest SkipHeaderVerification Tests" {
+        BeforeAll {
+            $Testfile = Join-Path $testdrive 'SkipHeaderVerification.txt'
+            'bar' | Set-Content $Testfile
+        }
 
-    It "Verifies Invoke-WebRequest default header handling with no errors" {
-        $uri = Get-WebListenerUrl -Test 'Get'
-        $headers = @{"If-Match" = "*"}
-        $response = ExecuteRequestWithCustomHeaders -Uri $uri -headers $headers
+        It "Verifies Invoke-WebRequest default header handling with no errors" {
```

Comment:
not sure whether these are tabs which have snuck in

Reply:
The section was moved from a `#region` to a `Context`, so indentation was added. see line https://github.com/markekraus/PowerShell/blob/eb541749b92cb5eb9f8c9a1d1658bed2af05f015/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1#L804